components/browser/icons/.gitignore

```diff
@@ -0,0 +1 @@
+manifest.json
```

Comment:
You mentioned there may be add-ons like `webcompat` that may not need the templating, do we want to make them use it inside AC?

Reply:
@pocmo Yes, so what I thought will do is apply this change to every project we want to. Here it's just for browser-icons, but we'd copy this to all projects, as needed. webcompat could decide to keep everything as it is.

I will talk to them to see if they want to keep versioning themselves. I will also see if I can make the gradle task reusable in simple way so we don't have to duplicate it (it's small, but still.)

Reply:
Makes sense. I also just noticed that this is the module `.gitignore` and not the root one. 👍

Reply:
@pocmo OK, I made the task re-usable now and applied it to all projects. The webcompat folks would like to continue using their own versioning as they already have scripts to export, plus they'd like to keep their versions in sync, which makes sense.

I have also added it to sample browser and the two test extensions to demo how this will look like if apps package their own extensions (e.g. Fenix and the two search telemetry extensions).